GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Ariane.cfg

```diff
@@ -268,34 +270,29 @@
     @manufacturer = Airbus Defence and Space
     @description = The second stage of the Ariane 5 launch vehicle family (ESC-A variant). Contains guidance capability.
 
-    @mass = 4.375
+    @mass = 4.45
     @crashTolerance = 14
     @breakingForce = 250
     @breakingTorque = 250
     @maxTemp = 773.15
     %skinMaxTemp = 873.15
 
-    @MODULE[ModuleRCS]
+    //  56 N Gas Hydrogen (GH2) attitude control thrusters for each axis (using liquid hydrogen directly for now).
+
+    @MODULE[ModuleRCS*]
     {
-        @thrusterPower = 0.4
+        @thrusterPower = 0.056
         !resourceName = NULL
 
         PROPELLANT
         {
-            name = Hydrazine
+            name = LqdHydrogen
             ratio = 1.0
         }
 
-        PROPELLANT
-        {
-            name = Nitrogen
-            ratio = 10.0
-            ignoreForIsp = True
-        }
-
         @atmosphereCurve
         {
-            @key,0 = 0 216
+            @key,0 = 0 180
```

Comment:
You may have to do something funny with this to account for the fact that it actually consumes GH2, while you have it consuming LH2. I'll have to think about that.